couchbase-sys/build.rs

```diff
@@ -3,6 +3,18 @@
 use std::env;
 use std::path::PathBuf;
 
+static ENV_FLAG_PREFIX: &'static str = "CB_";
+
+// Simple Environment Vairable filter
+fn environment_variable_filter(env_flag:&str) -> bool {
+    if env_flag.len() > ENV_FLAG_PREFIX.len() &&
```

Comment:
can this be simplified into just `env_flag.len() > ENV_FLAG_PREFIX.len() &&
       &env_flag[0..ENV_FLAG_PREFIX.len()] == ENV_FLAG_PREFIX` ? (avoid the if and return true false which is I think implicit in the logical and)

Reply:
Yeah, I can make the change. The only reason I had it in the if/else block was to allow for whitelisting and subsequent filtering. But can remove it for now.